### ISSUE TITLE
Otimizar figuras na página "Análises"

### ISSUE THREAD

**Issue by vsudbrack** [covid19br/covid19br.github.io#70]
Duas idéias que acho legal:

- [ ] Ao clicar sobre as imagens da página Análises elas abrem em popups maiores.

E olham para as imagens da página de nowcasting, elas ficam muuuuito grandes.

- [ ] Colocar a imagem em "meia coluna" com a legenda ao lado, reduzindo seu tamanho.

**Comment by renatocoutinho** [comment]
Esta página também podia ser uma lista de links (vertical) mesmo, em ordem cronológica

**Comment by AndreaSanchezTapia** [comment]
Só para precisar: em ordem cronológica inversa, do mais recente ao mais antigo.

**Comment by badain** [comment]
Concordo! Como já temos muitas análises a página está ficando gigante. O ideal seria tranformar `analises.html` em uma página de introdução e quebrar cada análise para um arquivo único em uma pasta `\analises`.

**Comment by AndreaSanchezTapia** [comment]
A outra coisa é que o link Análise leva ao primeiro artigo, que é o mais antigo. Mudando essa ordem talvez entraria direto na nota mais recente?
